FluentUI/build.gradle

```diff
@@ -2,88 +2,55 @@
-        versionCode 11
-        versionName '0.0.11'
+        minSdkVersion constants.minSdkVersion
+        targetSdkVersion constants.targetSdkVersion
+        versionCode 1
+        versionName "0.0.11"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-        vectorDrawables.useSupportLibrary = true
-    }
-    lintOptions {
-        abortOnError false
+        consumerProguardFiles 'consumer-rules.pro'
     }
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-    }
-    testOptions {
-        unitTests {
-            includeAndroidResources = true
-        }
-    }
-    productFlavors {
-    }
-}
-
-gradle.taskGraph.whenReady { taskGraph ->
-    taskGraph.allTasks.forEach {
-        if (it.name.contains("ReleaseUnitTest")) {
-            it.enabled = false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
-}
 
-repositories {
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
 }
 
 dependencies {
-    implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation 'com.android.support:appcompat-v7:28.0.0'
-    implementation 'com.android.support:exifinterface:28.0.0'
-    implementation 'com.android.support:recyclerview-v7:28.0.0'
-    implementation 'com.android.support:cardview-v7:28.0.0'
-    implementation 'com.android.support:design:28.0.0'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation project(':fluentui_core')
+    implementation project(':fluentui_listitem')
+    implementation project(':fluentui_drawer')
+    implementation project(':fluentui_tablayout')
+    implementation project(':fluentui_others')
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.jakewharton.threetenabp:threetenabp:1.1.0'
-    implementation 'com.splitwise:tokenautocomplete:2.0.8'
-    implementation "com.microsoft.device:dualscreen-layout:1.0.0-alpha01"
-    androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'org.robolectric:robolectric:4.0-alpha-3'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation "com.android.support:appcompat-v7:$supportVersion"
+    implementation "com.android.support:design:$supportVersion"
+    testImplementation "junit:junit:$junitVersion"
+    androidTestImplementation "com.android.support.test:runner:$testRunnerVersion"
+    androidTestImplementation "com.android.support.test.espresso:espresso-core:$espressoVersion"
 }
-
 task sourceJar(type: Jar) {
     from android.sourceSets.main.java.srcDirs
     classifier "sources"
 }
 
 publishing {
-    repositories {
```

Comment:
Are we not releasing maven now?

Reply:
We discussed this offline few days back that no one is using the packages from ADO feed

Reply:
I think we should not be removing this and continue publishing on our maven feed.
Biggest reason being Jcenter/bintray will be shutting down soon around May.  Only 2 months from now, many people within Microsoft suggest to use that only